Dockerfile

```diff
@@ -84,6 +84,15 @@ RUN \
 COPY ./docker/requirements.txt base_requirements.txt
 RUN pip install --disable-pip-version-check -U -r base_requirements.txt
 
+# Frontend builder
+
+FROM inventree_base as frontend
```

Comment:
Why don't you use the official node image as base?

Reply:
We need a full InvenTree environment for tags to work

Reply:
Sorry, which tags? vite build should run fine?

Reply:
Django template tags - we need those for plugins rendering panels

Reply:
Sorry, do you have an example of such a usage already?

Reply:
Nothing I have time to polish rn - plugins are not the focus of this PR.